web/solutii/chess table/index.html

```diff
@@ -0,0 +1,93 @@
+<head>
```

Comment:
Nu trebuie sa modifici un fisier in acest director. Trebuie respectata structura prezentat la laborator.
Structura corecta `web/solutii/<nume_prenume>/chess table/*` tu ai pus in root-ul `web/solutii/*`.

Pentru mai multe informatii acceseaza [pagina cursului](https://teaching.alexcoman.com/resurse/tutorial/git/2017/03/11/git-propunerea-unei-solutii/).